site/content/docs/5.3/components/navbar.md

```diff
@@ -516,7 +516,7 @@ Sometimes you want to use the collapse plugin to trigger a container element for
     <span class="text-body-secondary">Toggleable via the navbar brand.</span>
   </div>
 </div>
-<nav class="navbar bg-dark">
+<nav class="navbar">
```

Comment:
This one could maybe use a `data-bs-theme="dark"` because the content is also dark

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2345/commits/03882e8007d0a52095a92ad725abfa608e791f2e